examples/adder/cmd.go

```diff
@@ -184,16 +184,14 @@ var RootCmd = &cmds.Command{
 			PostRun: cmds.PostRunMap{
 				cmds.CLI: func(res cmds.Response, re cmds.ResponseEmitter) error {
 					clire := re.(cli.ResponseEmitter)
-
-					defer re.Close()
```

Comment:
Don't need to do this explicitly anymore. It's up to the executor to close the response emitter.